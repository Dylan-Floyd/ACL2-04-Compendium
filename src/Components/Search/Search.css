.search-div {
  display: inline-block;
}

.search-div > label > * {
  margin: 0px 2px;
}
