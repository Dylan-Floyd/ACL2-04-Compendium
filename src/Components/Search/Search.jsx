import React from 'react'

import './Search.css'

export default function Search({
  searchTerm,
  setSearchTerm,
  doSearch
}) {
  return (
    <div className='search-div'>
      <label>
        Search by Name:
        <input type='text' value={searchTerm} onChange={e => setSearchTerm(e.target.value)} />
      </label>
      <button onClick={doSearch}>Search</button>
    </div>
  )
}
