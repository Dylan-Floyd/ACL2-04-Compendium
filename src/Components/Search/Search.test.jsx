import { render } from '@testing-library/react'
import Search from './Search.jsx'

it('Should render the Search component', () => {
  const { container } = render(<Search searchTerm='bob' />)
  expect(container).toMatchSnapshot()
})