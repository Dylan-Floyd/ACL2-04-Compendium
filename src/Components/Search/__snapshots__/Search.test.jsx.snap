// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the Search component 1`] = `
<div>
  <div
    class="search-div"
  >
    <label>
      Search by Name:
      <input
        type="text"
        value="bob"
      />
    </label>
    <button>
      Search
    </button>
  </div>
</div>
`;
